boards/hifive1/Kconfig

```
@@ -17,3 +17,62 @@ config BOARD_HIFIVE1
     select HAS_PERIPH_SPI
     select HAS_PERIPH_TIMER
     select HAS_PERIPH_UART
+
+menu "Hifive1 clock configuration"
+
+choice "Clock source"
+prompt "Clock source selection"
+default CONFIG_USE_CLOCK_HFXOSC_PLL
+
+config CONFIG_USE_CLOCK_HFXOSC_PLL
```

The `CONFIG_` prefixes are added by Kconfig when generating the different outputs, it does not have to be added here. This would generate the macro `CONFIG_CONFIG_USE_CLOCK_HFXOSC_PLL` (check `bin/hifive1/generated/out.config`).
```suggestion
config USE_CLOCK_HFXOSC_PLL
```